core/vm/actor/builtin/types/miner/deadlines.cpp

```diff
@@ -11,10 +11,21 @@
 namespace fc::vm::actor::builtin::types::miner {
   using types::miner::kWPoStPeriodDeadlines;
 
+  static const auto kDeadlineIdError = ERROR_TEXT("invalid deadline id");
```

Comment:
if it's not visible outside this cpp file and used only once in loadDeadline, it can be put directly into loadDeadline (this macro creates error once with help of "static" local variable)